### ISSUE TITLE
Analysis hangs at "analysis still processing"

### ISSUE THREAD

**Issue by Bojak4616** [cuckoosandbox/cuckoo#1390]
tl;dr: Communication between the cuckoo host and the vm seems to be working fine. However, it appears the malware doesn't get executed (not sure where it gets placed either, can't determine if it is even dropped onto the vm). 

I'm pretty out of ideas on how to get this working, hopefully someone here can help. I am using Cuckoo 2.0.0 in venv with Virtualbox 5.1 on Linux version 4.4.0-72-generic (buildd@lcy01-17) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.4) ) Ubuntu SMP.
VM is WindowsXP SP3 32bit.

I followed documentation at https://cuckoo.sh/docs/.

Things I have tried:
* VM
  * Made sure logged in as administrator
  * Snapshot taken with with agent.py running
  * Agent.py in startup folder
  * Firewall turned off
* Host
  * Checked iptables rules (see pastebin below)
  * Disabled apparmor
  * Everything run with sudo to prevent permission issues
  * Checked cuckoo rooter settings (seems to be ok, though I know the least about this)

sudo cuckoo -d : https://pastebin.com/cjbUtP3A
cuckoo.conf : https://pastebin.com/Yx9YKEvb
virtualbox.conf : https://pastebin.com/URMtRFZS
What appears on the VM when a sample is submitted : http://i.imgur.com/vqA4hHm.png
- Also worth noting cuckoo files are created in the tmp directory created in C:\

**Comment by Bojak4616** [comment]
I've submitted multiple kinds of samples to see if it was just executables, still no luck. Will keep updated with other techniques I try in case someone runs into the same issue.

**Comment by Bojak4616** [comment]
Disabled the prompt asking if you are sure you want to run the executable, still no luck.

Gave up on Windows XP SP3 32bit. Moved over to Windows 7 64bit and it works fine. Guess I'll close the issue.
